framework/test/unit/modules/dpos_v2/utils.spec.ts

```diff
@@ -47,10 +47,13 @@ describe('utils', () => {
 			const expected: ModuleConfig = {
 				...defaultConfig,
 				minWeightStandby: BigInt(defaultConfig.minWeightStandby),
-				tokenIDDPoS: Buffer.from(defaultConfig.tokenIDDPoS, 'hex'),
+				governanceTokenID: Buffer.alloc(8),
```

Comment:
Probably, the same, would TOKEN_ID_LENGTH constant be a good idea, although this is test code.